src/attribution/onINP.ts

```diff
@@ -38,6 +38,12 @@ interface pendingEntriesGroup {
   entries: PerformanceEventTiming[];
 }
 
+/**
+ * The maximum number of LoAFs to retain on cleanup that don't have any
+ * overlapping events.
```

Comment:
Should we explain the rational for picking 50? We explain it below but maybe better pulling that comment up here?

Also this is not just "The maximum number of LoAFs", it's also the maximum number of Rendering times so maybe change this comment a little because of that?

Alternatively, mqake this just for LoAFs and add `LOAF` to the const name